docs/interpreter/livy.md

```diff
@@ -30,7 +30,7 @@ Additional requirements for the Livy interpreter are:
     <th>Description</th>
   </tr>
   <tr>
-      <td>zeppelin.livy.master</td>
+      <td>spark.master</td>
```

Comment:
Yes agreed. @mfelgamal can you prefix all properties with livy.* to maintain project styling WRT rest of the interpreters.